test/ext/staticarrays.jl

```diff
@@ -0,0 +1,27 @@
+using LinearAlgebra: mul!
```

Comment:
I would suggest putting this file in the `libraries` subsection. The only other thing you would need is make StaticArrays a test dep, but that's already the case.